### ISSUE TITLE
FerenOS StartPage

### ISSUE THREAD

**Issue by charliewalsh** [feren-OS/feren-os.github.io#10]
The feren os start page is down for me, It shows a unicorn, Is it the same for anything else?

**Comment by dominichayesferen** [comment]
Yeah, been working on turning it into a Chrome Extension, though progress is very slow on that due to work on the Store.

**Comment by charliewalsh** [comment]
On 2022-08-19 18:58, Dominic Hayes - Feren OS Dev wrote:
> Yeah, been working on turning it into a Chrome Extension, though
> progress is very slow on that due to work on the Store.
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe
> [2].
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> 
> Links:
> ------
> [1] 
> https://github.com/feren-OS/feren-os.github.io/issues/10#issuecomment-1220956493
> [2] https://github.com/notifications/unsubscribe

Okay then! Hope it works and good job on ferens design btw
